ee/api/test/test_team.py

```diff
@@ -52,27 +64,303 @@ def test_user_that_does_not_belong_to_an_org_cannot_create_a_project(self):
 
     # Deleting projects
 
-    def test_delete_team_own_second(self):
+    def test_delete_team_as_org_admin_allowed(self):
+        self.organization_membership.level = OrganizationMembership.Level.ADMIN
+        self.organization_membership.save()
+        response = self.client.delete(f"/api/projects/{self.team.id}")
+        self.assertEqual(response.status_code, HTTP_204_NO_CONTENT)
+        self.assertEqual(Team.objects.filter(organization=self.organization).count(), 0)
+
+    def test_delete_team_as_org_member_forbidden(self):
+        self.organization_membership.level = OrganizationMembership.Level.MEMBER
+        self.organization_membership.save()
+        response = self.client.delete(f"/api/projects/{self.team.id}")
+        self.assertEqual(response.status_code, HTTP_403_FORBIDDEN)
+        self.assertEqual(Team.objects.filter(organization=self.organization).count(), 1)
+
+    def test_delete_open_team_as_org_member_but_project_admin_forbidden(self):
```

Comment:
I wonder if project admins should have full permissions on their projects, including deleting them.